ddp-workspace/projects/ddp-dsm-ui/src/app/abstraction-field/abstraction-field.component.html

```diff
@@ -39,7 +39,7 @@
           'Abstraction-Background-Check': field.qcWrapper.check && (type === 'abstraction' || type === 'review'), 'Cell--Bottom-Border': field.type === 'medication' || field.type === 'recurrence'}">
       <b>{{field.displayName}}</b>
       <div *ngIf="participant[activityOfField] != null && participant[activityOfField].colorNotFinished && (!field.fieldValue.noData
-      && (field.fieldValue.value === ''|| field.fieldValue.value === null || field.fieldValue.value == undefined))"
+      && (field.fieldValue.value === ''|| field.fieldValue.value === null || field.fieldValue.value == undefined || field.fieldValue.value?.length === 0))"
```

Comment:
The part `field.fieldValue.value === null || field.fieldValue.value == undefined` 
can be changed like 

`field.fieldValue.value == null`